go/stellar/stellarsvc/service_test.go

```diff
@@ -991,6 +993,50 @@ func TestMakeAccountMobileOnlyOnRecentMobile(t *testing.T) {
 	checker.assertBundle(t, acctBundle, 4, 3, stellar1.AccountMode_USER)
 }
 
+func TestAutoClaimLoop(t *testing.T) {
+	tcs, cleanup := setupTestsWithSettings(t, []usetting{usettingFull, usettingFull})
+	defer cleanup()
+
+	acceptDisclaimer(tcs[0])
+
+	tcs[0].Backend.ImportAccountsForUser(tcs[0])
+	tcs[0].Backend.Gift(getPrimaryAccountID(tcs[0]), "100")
+	sendRes, err := tcs[0].Srv.SendCLILocal(context.Background(), stellar1.SendCLILocalArg{
+		Recipient:  tcs[1].Fu.Username,
+		Amount:     "3",
+		Asset:      stellar1.AssetNative(),
+		ForceRelay: true,
+	})
+	require.NoError(t, err)
+
+	acceptDisclaimer(tcs[1])
+	tcs[1].Backend.ImportAccountsForUser(tcs[1])
+	tcs[1].Backend.EnableAutoclaimMock(tcs[1])
+
+	tcs[1].G.GetStellar().KickAutoClaimRunner(tcs[1].MetaContext(), gregor1.MsgID{})
+
+	var found bool
+	for i := 0; i < 10; i++ {
+		time.Sleep(100 * time.Millisecond)
```

Comment:
maybe higher in CI? 

    100 * time.Millisecond * libkb.CITimeMultiplier(tcs[1].G)